modules/component/package.json

```diff
@@ -21,7 +21,8 @@
   "dependencies": {
     "@action-land/core": "^8.0.0",
     "@action-land/smitten": "^8.0.0",
-    "@action-land/tarz": "^8.0.0"
+    "@action-land/tarz": "^8.0.0",
+    "standard-data-structures": "^2.0.0"
```

Comment:
We might want to deprecate the usage of `linkedList` now that we have standard-data-structures.

Reply:
We can do that in separate PR ?